bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ApiConsumerHandler.java

```diff
@@ -87,40 +87,62 @@ private void initializeOnceBridgeOnline(FreeboxOsHandler bridgeHandler) {
-        }
-
         startRefreshJob();
     }
 
-    private void configureMediaSink(FreeboxOsHandler bridgeHandler, String upnpName) {
+    protected void configureMediaSink() {
         try {
+            String upnpName = editProperties().getOrDefault(Source.UPNP.name(), "");
             Receiver receiver = getManager(MediaReceiverManager.class).getReceiver(upnpName);
-            if (receiver != null && reg == null) {
-                ApiConsumerConfiguration config = getConfig().as(ApiConsumerConfiguration.class);
-                String callbackURL = bridgeHandler.getCallbackURL();
-                if (!config.password.isEmpty() || !receiver.passwordProtected()) {
-                    reg = bridgeHandler.getBundleContext().registerService(
-                            AudioSink.class.getName(), new AirMediaSink(this, bridgeHandler.getAudioHTTPServer(),
-                                    callbackURL, receiver.name(), config.password, config.acceptAllMp3),
-                            new Hashtable<>());
-                } else {
-                    logger.info("A password needs to be configured to enable Air Media capability.");
-                }
+            if (receiver != null) {
+                Map<String, String> properties = editProperties();
+                receiver.capabilities().entrySet()
+                        .forEach(entry -> properties.put(entry.getKey().name(), entry.getValue().toString()));
+                updateProperties(properties);
+
+                startAudioSink(receiver);
+            } else {
+                stopAudioSink();
             }
         } catch (FreeboxException e) {
             logger.warn("Unable to retrieve Media Receivers: {}", e.getMessage());
         }
     }
 
+    private void startAudioSink(Receiver receiver) {
+        FreeboxOsHandler bridgeHandler = checkBridgeHandler();
+        // Only video and photo is supported by the API so use VIDEO capability for audio
+        Boolean isAudioReceiver = receiver.capabilities().get(MediaType.VIDEO);
+        if (reg == null && bridgeHandler != null && isAudioReceiver != null && isAudioReceiver.booleanValue()) {
+            ApiConsumerConfiguration config = getConfig().as(ApiConsumerConfiguration.class);
+            String callbackURL = bridgeHandler.getCallbackURL();
+            if (!config.password.isEmpty() || !receiver.passwordProtected()) {
+                reg = bridgeHandler.getBundleContext()
+                        .registerService(
+                                AudioSink.class.getName(), new AirMediaSink(this, bridgeHandler.getAudioHTTPServer(),
+                                        callbackURL, receiver.name(), config.password, config.acceptAllMp3),
+                                new Hashtable<>());
+                logger.debug("Audio sink registered for {}.", receiver.name());
+            } else {
+                logger.info("A password needs to be configured to enable Air Media capability.");
```

Comment:
I would suggest warn level

Reply:
This is original code but I agree with you.

Reply:
Changed